kedro-airflow/RELEASE.md

```diff
@@ -1,6 +1,15 @@
 # Upcoming Release
 * Added support for Python 3.11
 
+# Release 0.6.1
```

Comment:
This heading can be removed and the new additions should be under the # Upcoming Release heading. We change these headings from "Upcoming Release" to "Release 0.6.x" when we do a release PR. Could you update this please? 😄

Reply:
Done!